nipype/utils/nipype_cmd.py

```diff
@@ -42,7 +42,11 @@ def run_instance(interface, options):
                     value = float(value)
                 except:
                     pass
-
+                # try to cast string input to boolean 
```

Comment:
there is a utility function to do this:

https://github.com/nipy/nipype/blob/master/nipype/utils/misc.py#L199